modules/auxiliary/scanner/rdp/cve_2019_0708_bluekeep.rb

```diff
@@ -4,6 +4,7 @@
 ##
 
 class MetasploitModule < Msf::Auxiliary
+  require 'rc4'
```

Comment:
I would put this `require` at the file level, though it doesn't matter practically speaking.

Reply:
Since this wasn't resolved, please use the suggested style next time. Thanks.